docs/faq.rst

```diff
@@ -41,10 +41,10 @@ Frequently asked questions
 **A:** You may export it with :ref:`export_ply_files` so that you could view it in Houdini or Blender.
        Or make use the extension library `Taichi THREE <https://github.com/taichi-dev/taichi_glsl>` to render images and update to GUI in real-time.
 
-**Q:** How do I declare a tensor with **dynamic length**?
+**Q:** How do I declare a field with **dynamic length**?
 
-**A:** What you want may be the ``dynamic`` SNode, a kind of sparse tensor, see :ref:`dynamic`.
-       Or simply allocate a dense tensor large enough, and another 0-D tensor ``tensor_len[None]`` for length record.
+**A:** What you want may be the ``dynamic`` SNode, a kind of sparse field, see :ref:`dynamic`.
+       Or simply allocate a dense field large enough, and another 0-D field ``field_len[None]`` for length record.
```

Comment:
Just to confirm:
@yuanming-hu Does `dense field`/`sparse field` sounds reasonable to you?

Reply:
Sure! Thanks for asking.